### ISSUE TITLE
 there's a solution ?

### ISSUE THREAD

**Issue by Gasvash** [Megabyte918/MultiOgar-Edited#1343]
How to solve the error with the skins, in the agar.io client, or you can no longer with the update

**Comment by AlexHGaming** [comment]
client sided

**Comment by Gasvash** [comment]
@AlexHGaming  but in the client agar.io there is no solution: "(

**Comment by CodeFlush** [comment]
@Gasvash  Yes there is, 
Find a agar.io client that supports skin , 
I recommend https://github.com/Luka967/Cigar/issues, host it , 
if you can pay for hosting, I recommend using Hostinger, 
If you want free hosting use 000webhost .
000webhost is pretty shitty, but You can still run the server.
Upload skins to that client and play. That's the best solution.
Agar.io Skins has been broken , so that is the best solution I can think of.

**Comment by Agarian** [comment]
I mean, you could use [my client](https://agarian.github.io), but eh.

**Comment by vamosperu** [comment]
@Gasvash 
I had the same problem and replace the following line # 289 PacketHandler.js
delete -> skinName = (inner == "r")? this.getRandomSkin (): inner;

replace -> skinName = (inner == "r")? "%" + this.getRandomSkin (): "%" + inner;

I hope it helps you

**Comment by vamosperu** [comment]
@Agarian  Sorry to bother you. You can show the time in the leaderboard

**Comment by CodeFlush** [comment]
@Gasvash  CLose?

**Comment by ZfsrGhS953** [comment]
@CodeFlush You don't understand... 000webhost is a bad idea for hosting **anything**, they inject advertising banners into the page which annoys the users and interferes with the code. According to some reviews online, they also add redirects to their advertising pages. GitHub Pages is a way to go.

**Comment by AlexHGaming** [comment]
@ZfsrGhS953 I am understanding with what you are saying, but 000webhost is like the **free**only source for hosting php and being non HTTPS. I haven't seen github pages work with PHP on my perspective of view. Of course I have learnt from my mistake a couple of months back to go with github pages for a useless calculator, but that's just HTML and JS. Dang if only there could be 1 website which provides free web hosting with no sleep or ads or watermarks and language supportation.

**Comment by ZfsrGhS953** [comment]
@AlexHGaming Yes, PHP can't be ran in Github Pages, but I think that @CodeFlush was talking about hosting a custom client. PHP is not necessary for that - everything in Cigar can be converted to JS.

**Comment by ZfsrGhS953** [comment]
@AlexHGaming As for "being non HTTPS", what problems do you experience with using that on GitHub Pages?

**Comment by AlexHGaming** [comment]
@ZfsrGhS953 https blocks the connection because github pages force the website to be https. Making it secure makes it worse

**Comment by Agarian** [comment]
Exactly ^^^^^^

**Comment by ZfsrGhS953** [comment]
@AlexHGaming Just turn it off then.

**Comment by AlexHGaming** [comment]
@ZfsrGhS953 the problem with turning it off, is that you can't. It's enforced to. Although it does make the site more secure and safer, https thinks that an agario clone would be insecure

**Comment by ZfsrGhS953** [comment]
@AlexHGaming Apologies, I thought there's a way to disable HTTPS by changing your domain from the default one, but for some reason when I try to change it gives me an error.

**Comment by AlexHGaming** [comment]
Which is why github pages is bad on other sides.

**Comment by ZfsrGhS953** [comment]
@AlexHGaming Yep, I can certainly tell you have tried and failed to solve the following problem:

**Given:**
FreeDNS
GitHub Pages
**Do:**
Make them work together.

**Comment by SuperOP535** [comment]
0fees.us is a pretty good PHP webhost with unlimited bandwidth & storage & parked domains & subdomains & addon domains and free online file manager. the only bad thing is only 500000 visits per day..
